### PR TITLE
Updating permissions page

### PR DESCRIPTION
**Pull request by hflynn**
Adding Read-write groups requires update of permissions docs (see https://github.com/openmicroscopy/openmicroscopy/pull/3783). Have also taken the opportunity to tidy up and improve styling.

Another commit to come to add read-write to tables.

### CODE REVIEW

File: omero/sysadmins/server-permissions.txt [hunk 1]
```diff
@@ -66,46 +62,79 @@ The various permission levels are:
...
+      
+      -  A private :term:`Group owner` can see and control who the group
+         members are and can view their data.
+      -  As a :term:`Group member`, you will only ever be able to see your
+         own data.
+      -  This can be used for general data storage, access and analysis, but
+         has very limited collaboration potential other than for the
+         :term:`Group owner` to see other group members' data.
+      
+      **Potential use-cases of Private group:**
```
Comment (mtbc) [inline omero/sysadmins/server-permissions.txt]: not sure "use-case" should be hyphenated here

File: omero/sysadmins/server-permissions.txt [hunk 2]
```diff
@@ -136,20 +169,21 @@ Permissions on your and other users' data
...
--  Move data between groups, if you belong to more than one group.
+-  create projects and/or datasets
+-  import data
+-  delete data
+-  edit names and descriptions of images
+-  change rendering settings on images
+-  annotate images (rate, tag, add attachments and comments)
+-  de-annotate (remove annotations that you have added)
+-  use Regions of Interest (ROIs) (add, import, edit, delete, save and analyze 
+   with them)
```
Comment (mtbc) [inline omero/sysadmins/server-permissions.txt]: "with"?

File: omero/sysadmins/server-permissions.txt [hunk 3]
```diff
@@ -136,20 +169,21 @@ Permissions on your and other users' data
...
+-  run scripts
+-  Move data between groups, if you belong to more than one group
```
Comment (mtbc) [inline omero/sysadmins/server-permissions.txt]: after previous, unexpected initial captial

**Comment by joshmoore** [comment]
> **Make certain all the members understand that this allows anyone in the group to permanently delete any of the data before performing this action**

Certainly seems to get the point across.

**Comment by joshmoore** [comment]
> trusting every member of the group

Again, makes good sense.

**Comment by hflynn** [comment]
This should be ok now. I'll rebuild on staging once Travis is green.

**Comment by hflynn** [comment]
https://www.openmicroscopy.org/site/support/omero5.1-staging/sysadmins/server-permissions.html

**Comment by joshmoore** [comment]
Is https://www.openmicroscopy.org/site/support/omero5.1-staging/sysadmins/server-permissions.html#term-move-between-groups not perhaps "Y"? @mtbc

**Comment by mtbc** [comment]
At the moment, a non-admin must own the objects that they are chgrp'ing. This could easily be relaxed to, say, requiring that they have permission to delete the objects.

**Comment by hflynn** [comment]
Correct that only Admins can move data between groups if they don't own it, even in RW groups - confirmed by @mtbc and @pwalczysko

**Comment by hflynn** [comment]
@joshmoore Petr's testing doc does have a note to say the implementation for this is different in the clients than the server.

**Comment by will-moore** [comment]
Looks fine. Good to merge.

**Comment by hflynn** [comment]
Just waiting on code PRs being merged to confirm this is going out with 5.1.2.

**Comment by joshmoore** [comment]
Merging for 5.1.2.

**Pull request merged**